core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -935,8 +935,12 @@ private[group] class GroupCoordinator(
           producerId,
           producerEpoch,
           RecordBatch.NO_SEQUENCE,
-          requestLocal,
-          postVerificationCallback
+          // Wrap the callback to be handled on an arbitrary request handler
+          // thread when transaction verification is complete.
```

Comment:
Should we leave a comment that the requestLocal passed in is only for the case where we execute immediately?